src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -33,6 +33,7 @@ import { MGPOptional } from 'src/app/utils/MGPOptional';
 import { NextGameLoadingComponent } from '../../normal-component/next-game-loading/next-game-loading.component';
 import { ArrayUtils } from 'src/app/utils/ArrayUtils';
 import * as Firestore from '@angular/fire/firestore';
+import { serverTimestamp } from 'firebase/firestore';
```

Comment:
Tu l'as sans doute vu mais "Firestore is defined but never used" ligne 35

Reply:
fixed